tests/blackbox/src/common/config.py

```diff
@@ -21,6 +21,8 @@ class Config:
     test_cluster_auth_token: str  # Gardener test cluster authentication token.
     aicore_deployment_id_gpt4: str
```

Comment:
can this be cleaned up?

Reply:
Let us keep it here so we can rather easily switch between models.